assets/pages/new-contributions-landing/components/ContributionTextInput.jsx

```diff
@@ -82,7 +85,10 @@ NewContributionTextInput.defaultProps = {
   max: undefined,
   min: undefined,
   step: undefined,
+  pattern: '.*',
```

Comment:
Default should probably be `undefined`, so that the browser will use the proper default value.

Reply:
👍